From 6f24419fa798d278230b12bf9905e5075a64b616 Mon Sep 17 00:00:00 2001
From: Alexander Couzens <lynxis@fe80.eu>
Date: Sun, 29 Mar 2015 00:40:11 +0100
Subject: [PATCH 8/9] FIXUP
 usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI

---
 ...999-usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI.patch | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/target/linux/generic/patches-3.18/999-usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI.patch b/target/linux/generic/patches-3.18/999-usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI.patch
index dfb771a..e4d3348 100644
--- a/target/linux/generic/patches-3.18/999-usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI.patch
+++ b/target/linux/generic/patches-3.18/999-usb-serial-gadget-no-TTY-hangup-on-USB-disconnect-WI.patch
@@ -13,13 +13,13 @@ power-cycles or for other reasons resets the USB host controller.
 
 Signed-off-by: Kyösti Mälkki <kyosti.malkki@gmail.com>
 ---
- drivers/usb/gadget/u_serial.c | 5 +++++
+ drivers/usb/gadget/functions/u_serial.c | 5 +++++
  1 file changed, 5 insertions(+)
 
-diff --git a/drivers/usb/gadget/u_serial.c b/drivers/usb/gadget/u_serial.c
+diff --git a/drivers/usb/gadget/function/u_serial.c b/drivers/usb/gadget/function/u_serial.c
 index b369292..d156a28 100644
---- a/drivers/usb/gadget/u_serial.c
-+++ b/drivers/usb/gadget/u_serial.c
+--- a/drivers/usb/gadget/function/u_serial.c
++++ b/drivers/usb/gadget/function/u_serial.c
 @@ -1258,8 +1258,13 @@ void gserial_disconnect(struct gserial *gser)
  	gser->ioport = NULL;
  	if (port->port.count > 0 || port->openclose) {
-- 
2.3.3

